src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -447,6 +447,47 @@ IsCitusTableType(Oid relationId, CitusTableType tableType)
 }
 
 
+CitusTableType
+GetCitusTableType(Oid relationId)
+{
+	CitusTableCacheEntry *tableEntry = LookupCitusTableCacheEntry(relationId);
+
+	/* we are not interested in postgres tables */
+	if (tableEntry == NULL)
+	{
+		return ANY_CITUS_TABLE_TYPE;
```

Comment:
I don't think there is any performance impact, but I agree that it might be confusing for the future callers.

So, changed to pass `CitusTableCacheEntry` instead of oid